staging/src/k8s.io/apimachinery/pkg/apis/meta/v1beta1/register.go

```diff
@@ -41,8 +41,9 @@ func AddMetaToScheme(scheme *runtime.Scheme) error {
 		&TableOptions{},
 		&PartialObjectMetadata{},
 		&PartialObjectMetadataList{},
+		&APIGroupDiscoveryList{},
+		&APIGroupDiscovery{},
```

Comment:
@deads2k @lavalamp Just want to confirm this is in the right location. Our old types were registered as unversioned (https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/register.go#L78-L80), I think we want these new types to be versioned correct?

Reply:
I personally still don't think they should be beta, but I haven't been following the conversation.